doc/source/whatsnew/v0.19.0.txt

```diff
@@ -1478,3 +1478,4 @@ Bug Fixes
 
 - Bug in ``eval()`` where the ``resolvers`` argument would not accept a list (:issue:`14095`)
 - Bugs in ``stack``, ``get_dummies``, ``make_axis_dummies`` which don't preserve categorical dtypes in (multi)indexes (:issue:`13854`)
+- Bug in ``internals._try_coerce_args``, when setting ```Timedelta``` with ```ix```  (:issue:`14155`)
```

Comment:
Can you make this clearer for the user? (specify the user-visible effect of the bug, not the underlying reason)